### PR TITLE
Add CPU microcode definition on uploaded files and docs

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

File: docs/source/metainfo.rst [hunk 1]
```diff
@@ -205,6 +205,8 @@ Allowed Category Values
 +-----------------------------+-----------------------------+
 | X-PlatformSecurityProcessor | Platform Security Processor |
 +-----------------------------+-----------------------------+
+| X-Microcode                 | CPU Microcode               |
```
Comment (hughsie) [inline docs/source/metainfo.rst:208]: Maybe an antisocial suggestion, but would `X-CpuMicrocode` be more accurate?

Comment (superm1) [reply docs/source/metainfo.rst:208]: I think you suggested this previously, but sure can adjust :)

Comment (hughsie) [reply docs/source/metainfo.rst:208]: Mea culpa.

**Comment by superm1** [comment]
I think you have to actually add category to database for this one when you merge though, right?

**Comment by hughsie** [comment]
> I think you have to actually add category to database for this one when you merge though, right?

Just done.

**Pull request merged**